dash-pipeline/Makefile

```diff
@@ -1,20 +1,27 @@
 SHELL = /bin/bash
 
+HAVE_DPAPP ?=
+ifeq ($(HAVE_DPAPP),y)
+DPAPP_LINK = veth4
+DPAPP_LINK_PEER = veth5
+SWITCH_PORT_DPAPP = --interface 2@$(DPAPP_LINK)
+endif
+
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 mkfile_dir := $(dir $(mkfile_path))
 
 # "All" type targets for convenience
-all:p4 sai saithrift-server docker-saithrift-client docker-saichallenger-client test
+all:p4 sai dpapp saithrift-server docker-saithrift-client docker-saichallenger-client test
```

Comment:
will be better to postpone the dpapp target in make all.

Reply:
> will be better to postpone the dpapp target in make all.

done